examples/run_composer_trainer.py

```diff
@@ -43,15 +43,16 @@ def main() -> None:
 
     trainer = hparams.initialize_object()
 
-    # Log the config to an artifact store
-    with tempfile.NamedTemporaryFile(mode="x+") as f:
-        f.write(hparams.to_yaml())
-        trainer.logger.file_artifact(LogLevel.FIT,
-                                     artifact_name=f"{trainer.logger.run_name}/hparams.yaml",
-                                     file_path=f.name,
-                                     overwrite=True)
-
-    # Print the config to the terminal on each local rank 0
+    # Log the config to an artifact store if global rank 0
```

Comment:
I'm being a real nit lol

```suggestion
    # Only log the config once, since it should be the same on all ranks.
```